pkgs/shells/dash/default.nix

```
@@ -52,5 +54,13 @@ stdenv.mkDerivation rec {
 
   passthru = {
     shellPath = "/bin/dash";
+    tests = {
+      "execute-simple-command" = runCommand "${pname}-execute-simple-command" { } ''
+        mkdir $out
+        ${dash}/bin/dash -c 'echo "Hello World!"' > $out/success
```

```suggestion
        ${dash}/bin/dash -c 'echo "Hello World!" > $out/success'
```

Might as well use shell redirection from within dash

---

Might as well :)